### ISSUE TITLE
Images do not open when right-clicking on an object

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#5]
```
What steps will reproduce the problem?
1. Right-Click on an item that has a local image attachment and its 
filename is not its UUID
2. Click open.

What is the expected output? What do you see instead?
The image should open in a viewer.

CAUSE:
Image handler is not renaming or using the path found in the imagelog.txt

WORKARAOUND:
  #Right-Click, select "Copy Asset UUID"
  #Rename the image using the Asset's UUID.  For example, change "Light 
Eyes.bmp" to "80fcc4c8-d202-cfbf-e772-c00feeceabc2.bmp"
  #Right-click > Open
```

Original issue reported on code.google.com by `joesox` on 23 Nov 2007 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
FIX:
Expected fix to be included with next release
```

Original comment by `joesox` on 23 Nov 2007 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
This issue has been FIXED in version 2.10.16
```

Original comment by `joesox` on 23 Jan 2009 at 7:43
- Changed state: **Verified**
